Makefile

```diff
@@ -63,7 +63,7 @@ test: ce  ## Runs the tests
 .PHONY: static-checks
 static-checks: ce  ## Runs all the static tests
 	$(VIRTUALENV)/bin/mypy --install-types --ignore-missing-imports bin lambda
```

Comment:
for consistency sake this line should probably also be changed to `bin/lib bin/test`

Reply:
Done - I extracted a variable too